src/lib/zork/index.ts

```diff
@@ -272,7 +270,6 @@ const onDataChannel = (ctx: Context, channel: any) => {
   // it forwards data for the SOCKS session.
   channel.onopen = () => ctx.log(`[channel ${channelId}] datachannel opened`);
   initSocksSessionAndFwdSocket(ctx, channel);
-  channel.onmessage = (event: any) => ctx.socksSession.handleDataFromSocksClient(event.data);
```

Comment:
It looks like this line was the source of the issue: the ctx.socksSession reference was not referring to the intended session. This PR removes the ctx.socksSession (and ctx.socksServer) references altogether in preference to the references inherited by the closures that need them, which are always correct.